jwst/regtest/test_nircam_mtimage.py

```diff
@@ -18,3 +21,30 @@ def test_nircam_image_moving_target(rtdata, fitsdiff_default_kwargs):
     fitsdiff_default_kwargs["atol"] = 1e-5
     diff = FITSDiff(rtdata.output, rtdata.truth, **fitsdiff_default_kwargs)
     assert diff.identical, diff.report()
+
+@pytest.mark.parametrize("input_file",
+                         [
+                             'jw00634_nrcblong_mttest_tnotinrange_uncal.fits',
+                             'jw00634_nrcblong_no_mtt_uncal.fits',
+                             'jw00634_nrcblong_mttest_uncal.fits',
+                         ])
```

Comment:
To make the parametrization a little clearer, could you add an `ids` kwarg to the parametrize function?
```
ids=["midpt_not_in_mt_table_range", "no_mt_table", "with_mt_table"]
```
That way the test name will have the above, useful info, instead of the filename.